pkg/resolver/search.go

```diff
@@ -96,20 +98,46 @@ func (s *SearchResult) buildSearchQuery(ctx context.Context, count bool, uid boo
 	// Example query: SELECT uid, cluster, data FROM search.resources  WHERE lower(data->> 'kind') IN
 	// (lower('Pod')) AND lower(data->> 'cluster') IN (lower('local-cluster')) LIMIT 10000
 	//FROM CLAUSE
-	schemaTable := goqu.S("search").Table("resources")
-	ds := goqu.From(schemaTable)
-	//SELECT CLAUSE
-	if count {
-		selectDs = ds.Select(goqu.COUNT("uid"))
-	} else if uid {
-		selectDs = ds.Select("uid")
+	if s.input.Keywords != nil {
+		schemaTable := goqu.S("search").Table("resources")
+		jsb := goqu.L("jsonb_each_text(?)", goqu.C("data"))
+		ds := goqu.From(schemaTable, jsb)
+		if count {
+			selectDs = ds.Select(goqu.COUNT("uid"))
+		} else if uid {
+			selectDs = ds.Select("uid")
+		} else {
+			selectDs = ds.Select("uid", "cluster", "data", "key", "value")
```

Comment:
We don't need to return `key` and `value`.  We use those only for matching, those will be included in `data`.

Reply:
Okay wasn't sure about that, thanks